tests/unit/App.spec.ts

```diff
@@ -882,8 +886,8 @@ describe('App as a Draft IA with approved NR', () => {
 
   beforeAll(() => {
     // clear store
-    store.commit('tasks', [])
-    store.commit('mutateFilings', [])
+    rootStore.setTasks([])
+    filingHistoryListStore.mutateFilings([])
```

Comment:
So I tested this. It works fine with setFilings!

Just for a bit of backstory: the reason why mutateFilings exists is because setFilings doesn't simply call it. Moreover, mutateFilings is called multiple times in the code. This is why both exist and are separate. 

At one point I thought of changing the name of mutateFilings to something like setCurrentFilings. However, setCurrentFiling without the s also already exists. I didn't want to cause any confusion with the code, so I just left it as is.

Reply:
Nevermind. I'm looking at it again and I see that mutateFilings hasn't been called anywhere. My memory is betraying me. I'm just going to remove it 😁